tunneling/python/app.py

```diff
@@ -0,0 +1,35 @@
+import json
+import urllib
+
+import flask
+import requests
+from flask_cors import CORS
+
+app = flask.Flask(__name__)
+CORS(app)
+
+
+@app.route("/bugs", methods=["POST"])
+def bugs():
+    sentry_host = "sentry.io"
+    known_project_ids = ["123456"]
```

Comment:
We could move these up to the global namespace and mark them with a comment that they should be changed accordingly (could also mention that in a README as a configuration step).

Reply:
moved and mentioned in readme